fhavro-export/src/main/scala/bio/ferlab/fhir/etl/task/FhavroExporter.scala

```diff
@@ -33,9 +32,32 @@ class FhavroExporter(bucketName: String, releaseId: String, studyId: String, stu
       .forResource(request.`type`)
       .returnBundle(classOf[Bundle])
 
+    val fhirStudyBundle = fhirClient.search()
+      .forResource("ResearchStudy")
+      .where(ResearchStudy.IDENTIFIER.exactly().code(studyId))
+      .returnBundle(classOf[Bundle])
+      .execute()
+
+    //should only be one matched
```

Comment:
Je pense tu devrais verifier cette condition si t es pas certain.

Reply:
Ok, le mot `should` peut porter a confusion.. si on a plus de une etude ya un probleme. `Must`...